components.js

```diff
@@ -598,6 +598,11 @@ var components = {
 			"title": "Xojo (REALbasic)",
 			"owner": "Golmote"
 		},
+		"xeora": {
+			"title": "Xeora",
+			"require": "markup",
+			"owner": "xeora"
```

Comment:
Please set the owner to you Github username `"freakmaxi"`.

Once this is changed I think we're good to go!